src/fosslight_dependency/_package_manager.py

```
@@ -566,26 +567,24 @@ def parse_gradle_download_lines(stdout_text, package_manager_name=''):
+    return get_google_maven_url(mvnrepo_url, group_id, artifact_id, version)
+
+
+def get_google_maven_url(mvnrepo_url, group_id, artifact_id, version):
+    group_path = group_id.replace('.', '/')
+    pom_url = (f"https://dl.google.com/dl/android/maven2"
+               f"/{group_path}/{artifact_id}/{version}/{artifact_id}-{version}.pom")
+    try:
+        resp = requests.head(pom_url, timeout=5, allow_redirects=True)
+        if resp.status_code == 200:
+            return f"https://maven.google.com/web/index.html?#{group_id}:{artifact_id}:{version}"
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)